linera-core/src/client.rs

```diff
@@ -679,7 +675,7 @@ where
     #[tracing::instrument(level = "trace")]
     /// Obtains the epochs of the committees trusted by the local chain.
     pub async fn epochs(&self) -> Result<Vec<Epoch>, LocalNodeError> {
-        let committees = self.committees().await?;
+        let (_epoch, committees) = self.epoch_and_committees(self.chain_id).await?;
```

Comment:
That part definitely seems odd:
* We have a `epoch_and_commitees` that returns epoch and the committees but drop the epoch.
* Then the epochs themselves are extracted from the committees, not the dropped epoch!

Reply:
Yes, I think the current epoch should always be among the keys in the `committees` map. We may want to add a check that that's actually the case.

In general, I was thinking of moving some chain info consistency checks to a central place (`LocalNodeClient::handle_chain_info_query`?). But that's for another PR.